html/changelogs/tinycircuits.yml

```diff
@@ -0,0 +1,4 @@
+author: "Hate9"
```

Comment:
Remove this file please, it's automatically generated based off your PR's comment.